doc/rst/technotes/libraries.rst

```diff
@@ -269,6 +269,14 @@ requires the following:
 - ``Cython``
 - ``numpy``
 
+If you are on a system where libraries are built statically by default (e.g.
+not OSx), you will need to set the environment variable ``CHPL_LIB_PIC`` to
+``pic`` and rebuild your Chapel compiler.  This will cause the Chapel runtime
```

Comment:
`rebuild your Chapel compiler` isn't quite right. The compiler won't be rebuilt, it's only the 3p/runtime.